io.github.iSlammedMyKindle.godotGem.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.iSlammedMyKindle.godotGem
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: godotGemClient.x86_64
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --allow=bluetooth
+  - --device=all
+modules:
+  - name: desktopEntry
+    buildsystem: simple
+    build-commands:
+      - mkdir $FLATPAK_DEST/share
+      - mv * $FLATPAK_DEST/share
+    sources:
+      - type: dir
```

Comment:
please don't use `dir`. list the individual files and use `install` to install them. Also this should be part of the main module.